superset/config.py

```diff
@@ -451,7 +452,7 @@ class D3Format(TypedDict, total=False):
     "ALERT_REPORTS": False,
     "DASHBOARD_RBAC": False,
     "ENABLE_EXPLORE_DRAG_AND_DROP": True,  # deprecated
-    "ENABLE_ADVANCED_DATA_TYPES": False,
+    "ENABLE_ADVANCED_DATA_TYPES": True,
```

Comment:
Why was this set to False in 3.1? @cccs-RyanK

Reply:
Yep should remove these changes they are likely local config